src/webview/types/events.types.ts

```
@@ -0,0 +1,81 @@
+export interface ReadyMessage extends WebviewMessageBase {
+  readonly command: 'ready';
+}
+
+export interface InitMessage extends WebviewMessageBase {
+  readonly command: 'init';
+  readonly config?: TerminalConfig;
+  readonly terminals?: any[];
+  readonly activeTerminalId?: string;
+}
+
+export type WebviewMessage = 
```

ESLint

[error] Delete `·`

(prettier/prettier)